torq/umbrel-app.yml

```diff
@@ -46,26 +46,22 @@ gallery:
   - 3.jpg
   - 4.jpg
 releaseNotes: >-
-  New features:
-
-  * Users can sign and verify signatures
-
-  *  Vector monitoring is now ready (and users can sign using Torq to get started)
-
-  *  Channel point column in the channel stable
-
-  *  In app "Help chat"
-
-  Fixes:
-
-  *  Several UI fixes
-
-  *  Links to mempool, amboss and 1ml are working again
-
-  *  Fix events toggling on channel inspect page
+  This release includes:
+  
+  * Automation!
+  
+  Automation is finally here, or at least the first part of it. You can now automate the channel policy (fee rate etc.) and add or remove tags. We will very soon release more advanced actions and of course Rebalancing!
+  
+  * Detailed data import choices
```

Comment:
The current design/layout of the update modal in Umbrel is shown like so:
![Screenshot 2023-02-13 at 19 41 52](https://user-images.githubusercontent.com/1112574/218558250-1566ee91-cbb7-4ec5-ad00-de77c423b5e0.png)

Could you add an extra empty above this line to add some spacing?